tools/astarte_export/lib/astarte/export.ex

```diff
@@ -29,18 +29,25 @@ defmodule Astarte.Export do
   """
 
   @doc """
-    The export_realm_data/2 function required 2 arguments to export
+    The export_realm_data/2 function required 3 arguments to export
     the realm data into XML format.
     the arguments are
+    - db-host-and-port -> -> A string in the format "hostname:port",
+      used to set the environment variables for Cassandra database connection.
     - realm-name -> This is a string format of input
     - file      -> file where to export the realm data.
     - options   -> options to export the realm data.
   """
 
-  @spec export_realm_data(String.t(), String.t(), keyword()) ::
+  @spec export_realm_data(String.t(), String.t(), String.t(), keyword()) ::
           :ok | {:error, :invalid_parameters} | {:error, any()}
 
-  def export_realm_data(realm, file, opts \\ []) do
+  def export_realm_data(db_host_and_port, realm, file, opts \\ []) do
+    [db_host, db_port] = String.split(db_host_and_port, ":")
+
+    System.put_env("CASSANDRA_DB_HOST", db_host)
+    System.put_env("CASSANDRA_DB_PORT", db_port)
```

Comment:
We shouldn't rely on setting env vars from the code. As an example, what if a user has set `CASSANDRA_DB_HOST` already for another task? This would override it.
Relevant variables should be passed down in the code, as we do e.g. for the `file`